dashboard/test/controllers/lti_v1_controller_test.rb

```diff
@@ -3,10 +3,15 @@
 require "test_helper"
 
 class LtiV1ControllerTest < ActionDispatch::IntegrationTest
-  setup_all do
+  setup do
```

Comment:
Need to understand a bit more with `setup` vs. `setup_all`, but the latter doesn't work when I try to stub the method from the LTI controller. [This SO article](https://stackoverflow.com/questions/57489057/difference-between-def-setup-and-setup-do-in-minitest) suggests that `setup` does stuff for both this class and the parent class.

Reply:
Actually, it might make sense. I think `LtiV1Controller.any_instance.stubs()` needs to get invoked in each test, which `setup` is doing, and `setup_all` only gets invoked once?

Reply:
Yep! `setup_all` is for setup that needs to happen once, for all tests; `setup` is for setup that needs to happen for each test.